src/index.js

```diff
@@ -14,21 +14,124 @@ function createElement(tagName, props, ...children) {
       element.appendChild(child);
       return;
     }
+
     element.appendChild(document.createTextNode(child));
   });
 
   return element;
 }
 
-function render() {
+function render({
+  operand1 = 0, operand2 = 0, operator = '', result = 0,
+}) {
+  const calculatorElements = {
+    operand1,
+    operand2,
+    operator,
+    result,
+  };
+
+  function calculator() {
+    const operators = {
+      '+': operand1 + operand2,
+      '-': operand1 - operand2,
+      '*': operand1 * operand2,
+      '/': operand1 / operand2,
+    };
+
+    return operators[operator];
+  }
+
+  function handleClickNumber(number) {
+    const temp = { ...calculatorElements };
+    if (operator !== '') {
+      if (typeof operand2 === 'number') {
+        temp.operand2 = Number(
+          operand2.toString() + number.toString(),
+        );
+        temp.result = temp.operand2;
+
+        render(temp);
+        return;
+      }
```

Comment:
만약 이부분을 `calculatorElements` 없이 진행을 해야한다면, 이렇게 바꿀수 있을거같아요! 어때요? 훨씬 간단해 보이지 않나요?!
```suggestion
  function handleClickNumber(number) {
    if (operator !== '') {
      if (typeof operand2 === 'number') {
        const newOperand2 = Number( operand2.toString() + number.toString() );
        
        render({
          operand1, operand2: newOperand2, operator, result: newOperand2
        });
        
        return;
      }
```